concepts/list-methods/about.md

```diff
@@ -176,146 +176,27 @@ You can also provide `start` and `end` indices to search within a specific secti
 
 ## Making Copies
 
-Remember that variables in Python are labels that point to underlying objects and `lists` are _container_ objects that hold references to their collected items.
+Remember that variables in Python are labels that point to underlying objects and `lists` are _container_ objects that only hold references to their collected items.
```

Comment:
s/collected/contained/ ?

Reply:
dunno.  They feel interchangeable to me.  Do you have  a preference?

Reply:
"collected" feels weird to me. Containers don't collect things. A person might collect things and put them into a container. But containers *do* contain objects.

Reply:
Good point.  I think I used "collected" because a list is a _type_ of _collection_.  But weird is weird.  Let's go with contain.  😄